pkgs/applications/version-management/git-and-tools/git-crypt/default.nix

```diff
@@ -20,7 +22,7 @@ stdenv.mkDerivation rec {
   '';
 
   installPhase = ''
-    make install PREFIX=$out
+    make ENABLE_MAN=yes install PREFIX=$out
```

Comment:
This can probably be refactored completely like

```nix
installFlags = [ 
  "ENABLE_MAN=yes"
  "PREFX=${placeholder ''out''}"
];

postFixup = ''
  wrapProgram $out/bin/* --prefix PATH : $out/bin:${git}/bin:${gnupg}/bin
'';
```

I'm also not sure you can glob like this with `wrapProgram`.

Reply:
Yeah, good catch, the glob only happens to work because there's only a single binary.  I'll make it explicit.

Also thank you for pointing out the flags approach, I like that.  I had never even heard of `placeholder` before.

Reply:
`placeholder` is builtin inside nix to get the path to an output